apps/src/applab/applab.js

```diff
@@ -292,14 +307,70 @@ function adjustAppSizeStyles(container) {
 var drawDiv = function () {
   var divApplab = document.getElementById('divApplab');
   divApplab.style.width = Applab.appWidth + "px";
-  divApplab.style.height = Applab.appHeight + "px";
+  divApplab.style.height = Applab.footerlessAppHeight + "px";
   if (Applab.levelHtml === '') {
     // On clear gives us a fresh start, including our default screen.
     designMode.loadDefaultScreen();
     designMode.serializeToLevelHtml();
   }
+
+  if (studioApp.share) {
+    renderFooterInSharedGame();
+  }
 };
 
+function renderFooterInSharedGame() {
+  var divApplab = document.getElementById('divApplab');
+  var footerDiv = document.createElement('div');
+  footerDiv.setAttribute('id', 'footerDiv');
+  divApplab.parentNode.insertBefore(footerDiv, divApplab.nextSibling);
```

Comment:
It looks like you are already inserting this as a sibling (rather than a child) of divApplab. So maybe you can keep rendering it this way and just change the math back so that the divApplab element is still 480px tall?